src/core/stdcpp/typeinfo.d

```diff
@@ -108,8 +108,8 @@ else version (CppRuntime_Gcc)
 
     class type_info
     {
-        void dtor1();                           // consume destructor slot in vtbl[]
-        void dtor2();                           // consume destructor slot in vtbl[]
+        void dtor1() {}                         // consume destructor slot in vtbl[]
+        void dtor2() {}                         // consume destructor slot in vtbl[]
```

Comment:
The vtable on the D side needs dummy definitions to put into the first 2 vtable slots.

Reply:
O_o ... this is like 12 months out of date.